docs/bigquery/snippets.py

```diff
@@ -89,476 +97,556 @@ def do_something_with(_):
+    assert table.table_id == 'my_table'
+    # [END create_table]
 
     to_delete.insert(0, table)
 
 
-def test_table_reload(client, to_delete):
+def test_get_table(client, to_delete):
     """Reload a table's metadata."""
-    DATASET_NAME = 'table_reload_dataset_%d' % (_millis(),)
-    TABLE_NAME = 'table_reload_table_%d' % (_millis(),)
-    dataset = client.dataset(DATASET_NAME)
-    dataset.create()
+    DATASET_ID = 'get_table_dataset_%d' % (_millis(),)
+    TABLE_ID = 'get_table_table_%d' % (_millis(),)
+    dataset = bigquery.Dataset(client.dataset(DATASET_ID))
+    dataset = client.create_dataset(dataset)
     to_delete.append(dataset)
 
-    table = dataset.table(TABLE_NAME, SCHEMA)
-    table.friendly_name = ORIGINAL_FRIENDLY_NAME
+    table = bigquery.Table(dataset.table(TABLE_ID), schema=SCHEMA)
     table.description = ORIGINAL_DESCRIPTION
-    table.create()
+    table = client.create_table(table)
     to_delete.insert(0, table)
 
-    # [START table_reload]
-    assert table.friendly_name == ORIGINAL_FRIENDLY_NAME
+    # [START get_table]
     assert table.description == ORIGINAL_DESCRIPTION
-    table.friendly_name = LOCALLY_CHANGED_FRIENDLY_NAME
     table.description = LOCALLY_CHANGED_DESCRIPTION
-    table.reload()                  # API request
-    assert table.friendly_name == ORIGINAL_FRIENDLY_NAME
+    table = client.get_table(table)  # API request
     assert table.description == ORIGINAL_DESCRIPTION
-    # [END table_reload]
+    # [END get_table]
 
 
-def test_table_patch(client, to_delete):
+def test_update_table_simple(client, to_delete):
     """Patch a table's metadata."""
-    DATASET_NAME = 'table_patch_dataset_%d' % (_millis(),)
-    TABLE_NAME = 'table_patch_table_%d' % (_millis(),)
-    dataset = client.dataset(DATASET_NAME)
+    DATASET_ID = 'update_table_simple_dataset_%d' % (_millis(),)
+    TABLE_ID = 'update_table_simple_table_%d' % (_millis(),)
+    dataset = bigquery.Dataset(client.dataset(DATASET_ID))
     dataset.description = ORIGINAL_DESCRIPTION
-    dataset.create()
+    client.create_dataset(dataset)
     to_delete.append(dataset)
 
-    table = dataset.table(TABLE_NAME, SCHEMA)
-    table.friendly_name = ORIGINAL_FRIENDLY_NAME
+    table = bigquery.Table(dataset.table(TABLE_ID), schema=SCHEMA)
     table.description = ORIGINAL_DESCRIPTION
-    table.create()
+    table = client.create_table(table)
     to_delete.insert(0, table)
 
-    # [START table_patch]
-    assert table.friendly_name == ORIGINAL_FRIENDLY_NAME
+    # [START update_table_simple]
     assert table.description == ORIGINAL_DESCRIPTION
-    table.patch(
-        friendly_name=PATCHED_FRIENDLY_NAME,
-        description=PATCHED_DESCRIPTION,
-    )      # API request
-    assert table.friendly_name == PATCHED_FRIENDLY_NAME
-    assert table.description == PATCHED_DESCRIPTION
-    # [END table_patch]
+    table.description = UPDATED_DESCRIPTION
 
+    client.update_table(table, ['description'])  # API request
```

Comment:
This should be `table = client.update_table(...)`

Reply:
Fixed in e30c19a